src/prototype_rust_sdk/workflow_context.rs

```diff
@@ -25,6 +29,7 @@ pub struct WfContext {
     args: Vec<Payload>,
     am_cancelled: watch::Receiver<bool>,
     shared: Arc<RwLock<WfContextSharedData>>,
+    next_sequence_number: AtomicU32,
```

Comment:
Why is this atomic? It's only ever accessed from inside the `impl` with `&mut` receivers

Reply:
I wasn't sure it was bound to one thread, I just realized it's only `&mut` receivers, thanks.

Reply:
I also don't love using the same sequence for all IDs. It's probably much easier for users to debug workflows against history if they see timers go 1,2,3,4 instead of 1,5,8 etc.

It's not the biggest deal, but also should be quite easy to do here, and hopefully in node too.